docs/BulkImport.md

```diff
@@ -3,7 +3,7 @@
 There are two modes of $import supported today-
 
 1. Initial mode is intended to load FHIR resources into an empty FHIR server. Initial mode only supports CREATE operations and, when enabled, blocks API writes to the FHIR server.
-1. Incremental mode is optimized to load data into FHIR server periodically and doesn't block writes via API. It also allows you to load lastUpdated and versionId from resource Meta (if present in resource JSON).
+1. Incremental mode is optimized to load data into FHIR server periodically and doesn't block writes via API. It also allows you to load lastUpdated, or both lastUpdated and versionId, from resource Meta (if present in resource JSON). There are no performance differences between incremental and initial modes.  
 
```

Comment:
Xoriant in their run identified performance difference. Lets double click offline

Reply:
In my tests on hospital data with ~1B resources performed on correctly scaled FHIR server and database I saw zero performance differences. This is as expected, because slight differences in compute on FHIR server related to processing modes do not change the database t-log bottleneck, and this bottleneck identifies the performance of import.